utils/data_version_control.py

```diff
@@ -98,3 +102,63 @@ def version_new_data(filename_roots: List[str]) -> None:
     dvc_files = [f"{fname}.csv.dvc" for fname in filename_roots]
     add_csv_files_to_dvc(csv_files)
     add_and_commit_dvc_files_to_git(dvc_files)
+
+
+def git_checkout_folder(
+    tag_name: Optional[str] = None,
+    commit_hash: Optional[str] = None,
+    folder_name: str = "data",
+) -> None:
```

Comment:
Should this return something, perhaps a `Path` object which is a pointer to the checked out folder?

Reply:
I think it is fine as it is for now